tests/integration/hdf5/test_io.py

```diff
@@ -586,3 +593,38 @@ def test_can_read_file_old_version(self):
     def test_can_read_file_invalid_hdf5_file(self):
         # current file is not an HDF5 file
         self.assertFalse(NWBHDF5IO.can_read(__file__))
+
+    def test_read_nwb_method_path(self):
+        
+        # write the example file
+        with NWBHDF5IO(self.path, 'w') as io:
+            io.write(self.nwbfile)
+            
+        # test that the read_nwb method works
+        read_nwbfile = NWBHDF5IO.read_nwb(path=self.path)
+        self.assertContainerEqual(read_nwbfile, self.nwbfile)
+
+        read_nwbfile.get_read_io().close()
+        
+    def test_read_nwb_method_file(self):
+        
+        # write the example file
+        with NWBHDF5IO(self.path, 'w') as io:
+            io.write(self.nwbfile)
+            
+        import h5py
+        
+        file = h5py.File(self.path, 'r')
+        
+        read_nwbfile = NWBHDF5IO.read_nwb(file=file)
+        self.assertContainerEqual(read_nwbfile, self.nwbfile)
+
+        read_nwbfile.get_read_io().close()
+    
+    @unittest.skipIf(not HAVE_FSSPEC, "fsspec library not available")
```

Comment:
@rly does Codecove run this test when `fsspec` is specified as an optional requirement or would we need to add `fsspec` elsewhere too? I'm just wondering because codecov seems to report the s3 parts of the code changes as not covered above, even though this test should cover them.

Reply:
It does run this. the codecov "Files Changed" annotation seems to be inconsistent with their website: https://app.codecov.io/gh/NeurodataWithoutBorders/pynwb/pull/1979?dropdown=coverage&src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=NeurodataWithoutBorders

Reply:
Thanks @rly for the clarification.